src/api/greptime/v1/database.proto

```diff
@@ -47,14 +46,7 @@ message InsertExpr {
 
 message ObjectResult {
   ResultHeader header = 1;
-  oneof result {
-    MutateResult mutate = 2;
-    FlightDataRaw flight_data = 3;
-  }
-}
-
-message FlightDataRaw {
-  repeated bytes raw_data = 1;
+  repeated bytes flight_data = 2;
```

Comment:
It is, a flight_data equals to a recordbatch. Arrow Flight's server returns a stream of recordbatch.

Reply:
Got it. Do we plan to support streaming results? If the server returns a stream of batches, it'd be better to avoid collecting all batches in the memory.

Reply:
Yes, planned.